apollo/lib/security/__init__.py

```diff
@@ -58,8 +58,7 @@ def get_principals(enhanced_http_connection, session):
         authenticated_user = enhanced_http_connection.current_user
         if authenticated_user:
             if authenticated_user.has_changed_initial_password:
-                principals.append(Authenticated)
-                principals += [Human, f"user:{authenticated_user.id}"]
+                principals += [Authenticated, Human, f"user:{authenticated_user.id}"]
```

Comment:
Line length, which is why I recommended the line break.

Reply:
I Blame it on vim

Reply:
```
au BufNewFile,BufRead *.py
    \ set textwidth=79 |
    \ set colorcolumn=80 |
    \ highlight ColorColumn ctermbg=red |
```
![200](https://user-images.githubusercontent.com/5303534/94263114-db3fdf80-ff34-11ea-9d00-b88d30b2f5cb.gif)